internal/privatemessaging/message.go

```diff
@@ -19,18 +19,21 @@ package privatemessaging
 import (
 	"context"
 
+	"github.com/hyperledger/firefly/internal/data"
 	"github.com/hyperledger/firefly/internal/i18n"
 	"github.com/hyperledger/firefly/internal/log"
 	"github.com/hyperledger/firefly/internal/sysmessaging"
-	"github.com/hyperledger/firefly/pkg/database"
 	"github.com/hyperledger/firefly/pkg/fftypes"
 )
 
 func (pm *privateMessaging) NewMessage(ns string, in *fftypes.MessageInOut) sysmessaging.MessageSender {
 	message := &messageSender{
 		mgr:       pm,
 		namespace: ns,
-		msg:       in,
+		group:     in.Group,
```

Comment:
Where is `group` used?

Reply:
Thanks, this was due to an earlier branch where I separated out the `InData` - but then I reverted to using a full `MessageInOut`